.github/workflows/cmake.yml

```diff
@@ -34,10 +34,6 @@ jobs:
             name: Windows x86_64
             preset: windows-x64
 
-#          - os: windows-latest
```

Comment:
why not enabling it?

Reply:
Two reasons really:

1. Nobody is asking for it. We initially did this because we thought we needed it for the zdew builds. But it turns out that zdew has been packaging 64-bit zet since the beginning.
2. Maybe the build will be useful to somebody out there with a special need, but including it in our releases might lead to more confusion than anything else.